src/app/reporting/Engine.cpp

```diff
@@ -148,7 +148,8 @@ std::optional<Status> ValidateAttributeIsReadable(DataModel::Provider * dataMode
 }
 
 DataModel::ActionReturnStatus RetrieveClusterData(DataModel::Provider * dataModel, const SubjectDescriptor & subjectDescriptor,
-                                                  bool isFabricFiltered, AttributeReportIBs::Builder & reportBuilder,
+                                                  bool isFabricFiltered, bool allowsLargePayload,
```

Comment:
2 booleans in a row start being hard to maintain (easy to switch order). Time to use a bitflag or param struct?

Reply:
And this is why C++ needs named params 🤣😅

As we're already using a bitflag one layer in, that makes sense, updated